xbmc/utils/ActorProtocol.cpp

```diff
@@ -19,6 +19,7 @@
  */
 
 #include "ActorProtocol.h"
+#include "memory.h"
```

Comment:
Is this a system header? What is it for?

Reply:
Afaik it's the systemheader holding memset, memcpy, etc.
Given the fact that <memory> is already included in ActorProtocol.h now, I think this is not necessary at all.

Reply:
If so, it should be `<cstring>`. `<memory>` is C++ stuff and unrelated.

Reply:
yeah, my bad <memory> != <memory.h>.
In this case it's not unnecessary ofc.
After some short googling I found memory.h being something rather old and <cstring> seems to be the better choice.
Anyway we've been using "memory.h" in the ActorProtocl header before, so I won't block this PR from being merged because of this minor.

Reply:
@MaxKellermann would be super nice if you could correct this in the course of include cleanup but it's also not a blocker for me

Reply:
Will do.